ote/src/cljs/ote/views/main.cljs

```diff
@@ -245,9 +245,14 @@
        [:li
         [linkify "http://bit.ly/nap-palaute" (tr [:common-texts :navigation-give-feedback]) {:target "_blank"}]]
        [:li [linkify "https://github.com/finnishtransportagency/mmtis-national-access-point/blob/master/docs/api/README.md" "Developers"]]
+       [:li [linkify "https://s3.eu-central-1.amazonaws.com/ote-assets/Tietosuojaseloste_NAP.pdf" (tr [:common-texts :navigation-privacy-policy]) {:target "_blank"}]]
        [:li
-        [language-selection e! style-base/language-selection-footer nil true]]]]]]])
-
+        [language-selection e! style-base/language-selection-footer nil true]]]]
+     [:div.col-md-12 (stylefy/use-style style-base/footer-copyright)
+      [linkify "http://creativecommons.org/licenses/by/4.0/"
+       [:img (merge (stylefy/use-sub-style style-base/footer-copyright :logo)
+                    {:src "/cc.svg"})] {:target "_blank"}]
```

Comment:
Not a biggie, but just one comment to make sure, that I'm reviewed this :)
Map that contains :target key, could be in the next row. It would be easier to see that it is a linkify parameter.
(and same thing few rows above)